addons/resource/__manifest__.py

```diff
@@ -13,7 +13,7 @@
 work center on manufacturing orders). This module manages a resource calendar
 associated to every resource. It also manages the leaves of every resource.
     """,
-    'depends': ['base', 'web'],
+    'depends': ['base', 'web', 'mail'],
```

Comment:
> Coucou @tivisse 🙂
> 
> We added `mail` module in the dependencies of resource module to be able to have
> 
> ```js
>     Many2ManyTagsAvatarUserField,
>     many2ManyTagsAvatarUserField,
>     Many2ManyAvatarUserTagsList,
>     Many2OneAvatarUserField,
>     many2OneAvatarUserField,
> ```
> 
> those components defined in mail module to extend them for `resource.resource` module. In that case, when we have resource module installed, we can have wonderful popover when we click on the avatar when `resource.resource` is used as many2one field in other model. 🙂
> 
> Does it okay for you or you prefer to avoid adding `mail` module in the dependencies of resource module?
> 
> Alternatively, we could move the code done in `resource` module into `hr` module.
> 
> cc @odoo/rd-hr

So as discussed, I've moved the avatar card code related to resource in resource_mail. If in the future the original code for avatar card is moved from mail to web, this bridge module could be removed and all the code put back into resource module.